pages/studentPortal/labs/multiplication-connect/Index.tsx

```diff
@@ -1,12 +1,21 @@
+import { indexOf } from "lodash";
 import React, { FC, useState, useEffect } from "react";
 import DiceSection from "../../../../components/math/multiplicationConnect/DiceSection";
 import GameBoard from "../../../../components/math/multiplicationConnect/GameBoard";
 import GameBoardBlock from "../../../../components/math/multiplicationConnect/GameBoardBlock";
 import PlayerSection from "../../../../components/math/multiplicationConnect/PlayerSection";
 import { getRandomItemFromArray } from "../../../api/random";
 
-export const calculateWinner = (grid: GameBoardBlock[]) => {
-  // Algorithm ran on block click to determine win
+enum selectedBy {
```

Comment:
Ah I see. You actually have a type called selectedBy.

I would capitalize the s here